### ISSUE TITLE
False Positive: F821(Undefined name) Error with `Literal`

### ISSUE THREAD

**Issue by hawk-tomy** [astral-sh/ruff#13068]
```py
from typings import Literal

type AllowedStringType = Literal['Allowed1','Allowed2']
```

F821 error on `Allowed1` and `Allowed2`.
[playground](https://play.ruff.rs/468a5656-f6f9-42a3-9f46-bb707aeb83b4)

**Comment by MichaReiser** [comment]
Can you try changing the import from `typing`**s**` to `typing`? 

https://play.ruff.rs/0fc19fc9-330b-42a3-968e-b819bc2bb153

**Comment by hawk-tomy** [comment]
It was my mistake. Thank you.
